lib/forms/pegasus_form_validation.rb

```diff
@@ -1,4 +1,5 @@
 require_relative 'pegasus_form_errors'
+require 'validates_email_format_of'
```

Comment:
[Sure is](https://github.com/validates-email-format-of/validates_email_format_of/blob/master/lib/validates_email_format_of.rb)